src/main/java/ladder/view/ResultView.java

```diff
@@ -17,65 +17,65 @@
   private static final String VERTICAL_LINE = "|";
 
   public void printLadder(LadderGame ladderGame) {
     System.out.println(RESULT);
-    System.out.println(printName(ladderGame.getGameInfo().users()));
-    String ladder = printLadderDetail(ladderGame.ladder());
+    System.out.println(printName(ladderGame.getGameInfo().getUsers()));
+    String ladder = printLadderDetail(ladderGame.getLadder());
     System.out.println(ladder.substring(0, ladder.length() - 1));
-    System.out.println(printPrize(ladderGame.getGameInfo().prizes()));
+    System.out.println(printPrize(ladderGame.getGameInfo().getPrizes()));
   }
 
 
   private String printLadderDetail(Ladder ladder) {
-    return ladder.lines()
-        .map(line -> printPoints(line) + System.lineSeparator())
+    return ladder.getLines()
+        .map(line -> EMPTY + printPoints(line) + System.lineSeparator())
         .collect(Collectors.joining());
   }
 
+
   private String printPoints(Line line) {
-    return line.points()
-        .map(point -> printUsedOrNot(point) + VERTICAL_LINE)
+    return line.getPoints()
+        .map(point -> VERTICAL_LINE + printUsedOrNot(point))
         .collect(Collectors.joining());
   }
 
   private String printUsedOrNot(Point point) {
-    if (point.isUsed()) {
+    if (point.isDirectionRight()) {
       return HYPHEN;
     }
     return EMPTY;
   }
 
-
   private String printPrize(Prizes prizes) {
-    return prizes.prizes()
-        .map(prize -> prize.prize())
-        .map(prize -> String.format(NAMES_FORMAT, prize))
+    return prizes.getPrizes()
+        .map(prize -> prize.getPrize())
+        .map(prize -> String.format(NAME_AND_PRIZE_FORMAT, prize))
         .collect(Collectors.joining());
   }
 
   private String printName(Users users) {
-    return users.users()
-        .map(user -> user.name())
-        .map(name -> String.format(NAMES_FORMAT, name))
+    return users.getUsers()
+        .map(user -> user.getName())
+        .map(name -> String.format(NAME_AND_PRIZE_FORMAT, name))
         .collect(Collectors.joining());
   }
 
   public void printAllResult(PrizeResult prizeResult) {
-    prizeResult.result()
+    prizeResult.getPrizeResult()
         .entrySet()
         .forEach(result -> System.out.println(
-            result.getKey().name() + " : " + result.getValue().prize()));
+            result.getKey().getName() + " : " + result.getValue().getPrize()));
   }
 
   public void printEachResult(PrizeResult prizeResult, String userName) {
     System.out.println(RESULT);
     String result = "";
-    for (Entry<User, Prize> prizeMap : prizeResult.result().entrySet()) {
-      if (prizeMap.getKey().name().equals(userName)) {
-        result = prizeMap.getValue().prize();
+    for (Entry<User, Prize> prizeMap : prizeResult.getPrizeResult().entrySet()) {
+      if (prizeMap.getKey().getName().equals(userName)) {
+        result = prizeMap.getValue().getPrize();
         break;
       }
```

Comment:
```suggestion
    User user = new User(userName);
    Prize prize = prizeResult.get(user);
    if (prize != null) {
      result = prize.getPrize();
    }
```
Map을 사용한 이점을 보지 못하고 있네요. Map을 적절히 사용하면 탐색을 효율적으로 진행할 수 있어요.
성능도 좋아지고 코드 가독성에도 도움이 됩니다. 🙂

Reply:
그렇군요 좋은 의견 감사합니다..!